### ISSUE TITLE
[med] JAWS+Chrome: Many Pages: User not notified of text input description when displayed

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5659]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has contacts or recent chats that display in the new group panel.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Tab to focus the "+" button and press Enter key.
3. Tab to focus "New Group" option and press Enter key.
4. Enter any single digit (e.g. 1) into the search field and note what is displayed and announced.

## Expected Result:
Once the field validation determines a number has been entered, the user is notified of the format required for inputting a phone number into the search field with text below the field. The text should be announced when displayed. i..e. "Please enter a phone number including the country code e.g +447814266907".

## Actual Result:
The formatting requirement text is displayed below the field but never announced.

HTML:
`<div class="css-1dbjc4n" style="padding-bottom: 20px; padding-right: 20px; padding-left: 20px;"><div dir="auto" class="css-901oao" style="color: rgb(125, 139, 143); font-family: GTAmericaExp-Regular; font-size: 13px; line-height: 18px;">Please enter a phone number including the country code e.g +447814266907</div></div>`

Other occurrences
- The same issue is seen for Search overlay on Chat page.
- Reproduced for the search field on New Chat overlay.

## Workaround:
No workaround

## Area issue was found in:
Create new group overlay, Chat page, Create new chat overlay

## Failed WCAG checkpoints
4.1.3

## User impact:
Screen reader users will have no idea that phone number input has been detected and additional input info is provided.

## Suggested resolution:
Use aria-live="polite" attribute with the text container so changes are announced.
https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5186882_Chrome_Jaws-Chat-User_not_notified_of_text_input_description_when_displayed](https://user-images.githubusercontent.com/88733897/136069138-73c0052f-4d53-4c90-9e84-b2a82b7a0d0e.png)
![Bug5186882_Chrome_Jaws-Create_New_Group-User_not_notified_of_text_input_descrition_when_displayed](https://user-images.githubusercontent.com/88733897/136069152-d5140a14-98b1-4b9f-b4d1-56325fe52743.png)

https://user-images.githubusercontent.com/88733897/136069158-2c4cd62e-0cb6-4f6b-956a-e71caab15192.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
